resources/js/Pages/Welcome.jsx

```diff
@@ -1,6 +1,8 @@
 import { Link, Head } from '@inertiajs/react';
 import logo from '../../images/Mainlogo.png';
 import GithubLogo from '../../images/dashboard/github.svg';
+import Button from '../Components/Button/Button';
```

Comment:
instead of `../` you can use `@/Components`

instead of `'../../images/` you can use `~/images`